### PR TITLE
Favourites

### PR DESCRIPTION
**Pull request by azhara-a**
Add a page for a finished recipe with 2 buttons:
1. Save to favourites: sends a post request to `/api/favourites` with the recipeId to add it to user's list of favourite recipes
2. It was alright: redirects to `/home`

The user interface will be changed in a follow up CL
image

### CODE REVIEW

File: backend/src/main/java/utils/RecipeConstants.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.utils;
+
+public final class RecipeConstants {
```
Comment (mam249) [inline backend/src/main/java/utils/RecipeConstants.java:17]: What's the purpose of this file? You haven't used it anywhere afaik

**Review by mam249 — COMMENTED** [review]
LGTM apart from me not getting why you have recipeconstant

File: backend/src/main/java/utils/RecipeConstants.java (hunk 1 above)
Comment (azhara-a) [reply backend/src/main/java/utils/RecipeConstants.java:17]: Accidentally added it

**Review by lminchillo — APPROVED** [review]
LGTM!

**Pull request merged**